src/pages/Login.tsx

```diff
@@ -43,30 +43,29 @@ export default function Register() {
 						</ul>
 
 						<form ref={formRef} onSubmit={onLogin}>
-							<fieldset class="form-group" aria-required>
+							<fieldset class="form-group">
 								<input
-									value={form.email}
 									class="form-control form-control-lg"
 									type="email"
-									required
 									placeholder="Email"
+									required
+									value={form.email}
 									onInput={e => setForm(prev => ({ ...prev, email: e.currentTarget.value }))}
 								/>
 							</fieldset>
 							<fieldset class="form-group">
 								<input
-									value={form.password}
 									class="form-control form-control-lg"
 									type="password"
-									required
 									placeholder="Password"
+									required
+									value={form.password}
 									onInput={e => setForm(prev => ({ ...prev, password: e.currentTarget.value }))}
 								/>
 							</fieldset>
 							<button
 								class="btn btn-lg btn-primary pull-xs-right"
 								disabled={!form.email || !form.password}
-								type="submit"
```

Comment:
I want to verify whether I can tap <kbd>Enter</kbd> in password input filed will submit the form. (I don't remember the submit button can do this or not), but I found that the ONLINE page was broken. 😂

Reply:
Submit is the default behavior in a form. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type

Reply:
`type="submit"` is default within a form, so this does nothing: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type

Pressing `Enter` works just as you'd expect.